example/ios/Podfile

```diff
@@ -18,7 +18,7 @@ target 'DdSdkReactNativeExample' do
     :app_path => "#{Pod::Config.instance.installation_root}/.."
   )
 
-  pod 'DatadogSDKReactNative', :path => '../../packages/core/DatadogSDKReactNative.podspec', :testspecs => ['Tests']
+  # pod 'DatadogSDKReactNative', :path => '../../packages/core/DatadogSDKReactNative.podspec', :testspecs => ['Tests']
```

Comment:
should it be reverted?

Reply:
This one should be commented, we copy the `PodfileForTests` that contains this line to enable tests on the CI. 
By keeping this line, we break the `pod install` command for the example project.